client/src/pages/workspace/Workspace.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {useState} from "react";
+import Board from "./board/board";
+import {BoardEntity} from "../../model/entity/BoardEntity";
+import {List, ListItem} from "@mui/material";
+import AddBoard from "./board/AddBoard";
+
+const Workspace: React.FC = () => {
+    const [boards, setBoards] = useState<BoardEntity[]>([
+        {id: 1, name: 'white board', description: 'mega ultra super cool red board', createdAt: new Date(), updatedAt: new Date()},
+        {id: 2, name: 'board2', description: 'boardDesc2', createdAt: new Date(), updatedAt: new Date()}
+    ]);
+
+    const addBoard = (newBoard: BoardEntity) => {
+        setBoards([...boards, newBoard]);
+    }
+
+    const deleteBoard = (id: number) => {
+        const updatedBoards = boards.filter(board => board.id !== id);
+        setBoards(updatedBoards)
+    }
+
+    return (
+        <div style={{verticalAlign: 'middle'}}>
+            <List style={{display: 'inline-flex'}}>
```

Comment:
maybe you can remove `<div>` and use style in `<List>`
`<List style={{display: 'inline-flex', verticalAlign: 'middle'}}>`

Reply:
vertical align doesnt work anyway, i just forgot to delete it